### ISSUE TITLE
shutdown_switch_enabled

### ISSUE THREAD

**Issue by wageck** [MatiasTK/PlexAutoShutdown#2]
Is it Possible to set shutdown_switch_enabled = True with some start argument or move it in the Config File ?

**Comment by Yw0ke** [comment]
Yeah, some kind of way to launch this with task scheduler at a specific time and have the toggle on by default would be great.
Good job on the project tho, very niche, i like it <3
